### ISSUE TITLE
IDRAC9 get local user list

### ISSUE THREAD

**Issue by sysadm24** [dell/dellemc-openmanage-ansible-modules#401]
#### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) 
  to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, 
  they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

**Comment by anupamaloke** [comment]
@sysadm24, you should be able to get a list of all local users by making a direct call to iDRAC REST API using the `ansible.builtin.uri` module.

 ```
  tasks:
  - ansible.builtin.uri:
      url: "https://{{ inventory_hostname}}/redfish/v1/AccountService/Accounts?$expand=*"
      user: "{{ user }}"
      password: "{{ password }}"
      method: "GET"
      status_code: 200
      validate_certs: no
      force_basic_auth: yes
      headers:
        Accept: "application/json"
```

**Comment by anupamaloke** [comment]
Closed by #458
